### ISSUE TITLE
Examples aren't working

### ISSUE THREAD

**Issue by rivertam** [jhdrn/myra#3]
Maybe I'm being dumb but just following the basic instructions for the examples gives this:

![screenshot_2016-40-16_23-40-23](https://cloud.githubusercontent.com/assets/4165105/17723806/dd72a5e0-640a-11e6-89c1-614e6a953e55.png)

I tried to figure it out, but I couldn't. :/

**Comment by jhdrn** [comment]
That's strange. Do you get the same error for all examples?

**Comment by rivertam** [comment]
I only tried I think 3, but yes. :/ I think it's kind of a bug with typescript loader, but I'm not sure why we get different results.

**Comment by rivertam** [comment]
Pretty much all I found was this https://github.com/TypeStrong/ts-loader/issues/91

**Comment by jhdrn** [comment]
Seems like TS-loader does not emit Typescript errors. Do you get any errors if you compile the Typescript files manually by running: `./node_modules/.bin/tsc -p ./ --outDir tmp`?

**Comment by rivertam** [comment]
FYI I'm trying this on a new computer which is still having the issue. That command doesn't give any errors to stdout or stderr. :/

**Comment by jhdrn** [comment]
After running `tsc`, did you get output in the tmp folder?

I assume you get the same errors if you run `webpack`? What operating system are you running on?

**Comment by rivertam** [comment]
Yeah I get a src/ folder and a node_modules/ folder inside tmp.

And yes I get the same issues with `webpack`.

The computer I originally posted on was Linux/Ubuntu, and this one is OS X, so basically "both".

**Comment by jhdrn** [comment]
Very strange. I'm running on Windows 10 without any problems. I'll download a VirtualBox image of Ubuntu and try it myself...

**Comment by rivertam** [comment]
Okay great, thanks. I would help but I don't have all that much time to spare on this. I'd love to donate time in the future, though.

**Comment by jhdrn** [comment]
Ok, now I've tried it on Ubunto too. I got the same errors :/ 

By some reason does the Typescript compiler not emit javascript for some of the files. I'm not sure if it's a bug in the compiler or if it has something to do with the files themselves. It's very strange because when turning on tracing, I can see that the compiler finds the files, but it still doesn't compile them. I'll continue investigating...
